pages/common/kubectl.md

```diff
@@ -1,13 +1,9 @@
 # kubectl
 
 > Command line interface for running commands against Kubernetes clusters.
-> More information: <https://kubernetes.io/docs/tasks/tools/install-kubectl>.
+> More information: <https://kubernetes.io/docs/reference/kubectl/>.
 
-- List all pods in all namespaces:
-
-`kubectl get pods --all-namespaces`
-
-- List all pods with more information (such as node name):
+- List all information about a resource (pods, deployments, services, ingresses, etc.) with more details:
```

Comment:
We can remove (pods, deployments ..) from the description as we are going to mention those in the example

Reply:
I agree on that.